docs/index.rst

```diff
@@ -6,10 +6,18 @@ infrastructure to build ITK external module Python wheels.
 
 `ITK <https://www.itk.org/>`_ is an open-source, cross-platform system that provides developers with an extensive suite of software tools for image analysis.
 
-The Python packages are built daily. To install the ITK Python package::
+To install the stable ITK Python package::
+
+  python -m pip install --upgrade pip
+  python -m pip install itk
+
+The Python packages are built daily. To install the latest build from the ITK
+Git *master* branch::
+
+  python -m pip install --upgrade pip
+  python -m pip install itk \
+    -f https://github.com/InsightSoftwareConsortium/ITKPythonPackage/releases/tag/latest
```

Comment:
Adding `--no-index` would ensure it fails if the attempted action of installing nightly packages is indeed failing ... 

```
python -m pip install itk \
    -f https://github.com/InsightSoftwareConsortium/ITKPythonPackage/releases/tag/latest --no-index
```

Reply:
Great idea! :+1: